src/main/java/org/apache/rocketmq/dashboard/controller/ConsumerController.java

```diff
@@ -37,6 +38,7 @@
 
 @Controller
 @RequestMapping("/consumer")
+@Permission
```

Comment:
I still recommend using the unified configuration, which can be unified in one directory, without annotating each method.

Reply:
The @permission annotation is on a class in which all the methods of the class will implement aop methods. There is no need to annotate each method